MovieApp/MovieApp/App/Features/Movies/ViewControllers/DetailsViewController/DetailsViewController.swift

```diff
@@ -23,17 +25,34 @@ class DetailsViewController: UIViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
-
+        
         buildViews()
         presenter.setDelegate(delegate: self)
         presenter.fetchData()
+        
+        setupNavigationBackButton()
```

Comment:
no need for extra line and this should be called after buildViews() that way ytou have your views logic and after that data logic